src/PactNet.AspNetCore/PactNet.AspNetCore.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\PactNet.Native\PactNet.Native.csproj" />
+    <ProjectReference Include="..\PactNet\PactNet.csproj" />
+  </ItemGroup>
+
+  <ItemGroup Condition="$(TargetFramework.StartsWith('netstandard'))">
```

Comment:
Everything in here is .Net Standard so these 2 sections should be merged together. I'm not enturely sure we actually need any of these anyway?

Reply:
I think it might have been automatically added. No problem with the acceptance test.